config/crd/bases/airflow.apache.org_airflowclusters.yaml

```diff
@@ -695,6 +695,17 @@ spec:
                       description: Configure git to ignore ssl (for use with self-signed
                         certs), false to ignore
                       type: boolean
+                    sync-image:
```

Comment:
I don't think using `-` as a separator is in line with how fields are named in k8s in general - can you turn these into `syncImage` etc., please?

Reply:
@vpavlin done